content/review/content-management.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Content Management"
+metaTitle: "sensenet - Content Management"
+metaDescription: "sensenet Content Management"
+---
+
+# What does content management mean?
+
+For content management we mean the processes of storing, editing, organizing and delivering content.
+# What is content regarding to sensenet?
+In sensenet everything is content. Documents, tasks, users, projects, even system configuration files – everything – are stored and managed in the [content repository](/concepts/content-repository). It is the foundation that makes content management easy and powerful at the same time. All content items are organized in a [content tree](/concepts/content-tree), so you can always think in subtrees and inheritance when managing and presenting content.
+
+# Basic content management actions in sensenet
+
+All the basic content management actions are available in sensenet, such as:
```

Comment:
I would split these in two section:

basic content management actions
other additional actions
Preview, search, share are not basic actions in my opinion.

AND and maybe they could be explained a bit more

Reply:
a CRUD operationt is mondatba foglalni